extensions/spotify/src/main/java/app/revanced/extension/spotify/misc/fix/WebApp.java

```diff
@@ -264,4 +283,12 @@ private static void setCookies(@NonNull String cookies) {
             cookieManager.setCookie(OPEN_SPOTIFY_COM_URL, cookie);
         }
     }
+
+    public interface NativeLoginHandler {
+        void login();
+    }
+
+    public static void setNativeLoginHandler(NativeLoginHandler handler) {
```

Comment:
can be package private, apply on other members

```suggestion
    static void setNativeLoginHandler(NativeLoginHandler handler) {
```